src/ui/components/Video.tsx

```diff
@@ -58,7 +59,7 @@ function Video({
   const showCommentTool = isPaused && !isNodeTarget && isAuthenticated && !isNodePickerActive;
 
   return (
-    <div id="video">
+    <Redacted allowOptIn id="video">
```

Comment:
this is already redacted by virtue of the fact that LogRocket does not pick up canvas or video. And it would be nice to include the comment markers.